task-2/controllers/user-controller.ts

```diff
@@ -0,0 +1,106 @@
+import USERS_ from '../data/users.json';
+import { errorResponse } from '../helpers';
+import { User } from '../interfaces/user';
+import { passwordSchema, userCreateSchema, userUpdateSchema } from '../schemas/user.schema';
+import { v4 as uuidv4 } from 'uuid';
+
+const USERS = USERS_ as User[];
+
+const createUser = (req, res) => {
```

Comment:
`import { Request, Response } from 'express';

const createUser = (req: Request, res: Response)`
for correct types